### PR TITLE
Codecommit drill documentation update

### PR DESCRIPTION
**Pull request by minhngocd**
Updated documentation after running through drill instructions for Deploying via CodeCommit

https://trello.com/c/L46MTjI2/2369-2nd-line-drill

### CODE REVIEW

File: source/manual/github-unavailable.html.md [hunk 1]
```diff
@@ -18,8 +18,8 @@ build and deploy a container image from your workstation.
 Run the following commands from the root directory of the repository.
 
 1. [Set up CodeCommit on your machine](/manual/howto-checkout-and-commit-to-codecommit.html#install-dependencies-and-set-up-local-environment) if you haven't already.
-1. Follow the [CodeCommit guide](/manual/howto-checkout-and-commit-to-codecommit.html#quick-reference-guide) to clone the repo and commit/push changes if needed.
-1. Set the image tag, image registry and image repository names that you will use by running the following commands. You don't need to modify any of the values in the commands.
+2. Follow the [CodeCommit guide](/manual/howto-checkout-and-commit-to-codecommit.html#quick-reference-guide) to clone the repo and commit/push changes if needed.
```
Comment (davidgisbey) [inline source/manual/github-unavailable.html.md:21]: probs overkill but we could potentially mention that if it's a repo you've got in the `/govuk` dir you need to clone it somewhere else. Although i'm sure people will be able to figure that out

**Comment by davidgisbey** [comment]
It looks like there's some issues with trailing spaces etc. but this looks great. Really throughly explained 👍

**Review by davidgisbey — APPROVED** [review]
Thanks so much for writing this up. I just pushed up the changes to stop the linter complaining. Hope that's okay. I just wanted to save you some hassle since you're at lunch then have a meeting

File: source/manual/2nd-line-drills.html.md [hunk 2]
```diff
@@ -72,7 +72,29 @@ See [Communicate when Slack is unavailable](/manual/slack-unavailable.html#2nd-l
 
 ### Deploy from AWS CodeCommit when Github is unavailable
 
-Follow the [Deploy when GitHub is unavailable](/manual/github-unavailable.html#create-and-deploy-an-image-to-ecr) instructions.
+#### Warnings
+
+- Please run the following drill in `integration`. Some steps in the guide below refer to production poweruser, be mindful and choose the correct environment for the drill.
```
Comment (ChrisBAshton) [inline source/manual/2nd-line-drills.html.md:77]:
```suggestion
- Please run the following drill in `integration`. Some steps in the guide below refer to production, be mindful and choose the correct environment for the drill.
```

File: source/manual/2nd-line-drills.html.md [hunk 3]
```diff
@@ -72,7 +72,29 @@ See [Communicate when Slack is unavailable](/manual/slack-unavailable.html#2nd-l
...
+- Please choose a low impact deployable application that has a Dockerfile. The drill does not cater for repos that are dependencies such as gems and npm packages.
```
Comment (ChrisBAshton) [inline source/manual/2nd-line-drills.html.md:78]: I don't think we need to refer to gems/NPM packages here? This drill is very much scoped to deploying GOV.UK applications when GitHub is down.

Rubygems/NPM have their own separate deployment pipelines and I can't think of a situation where we'd need to deploy either of those during a GitHub incident.

File: source/manual/2nd-line-drills.html.md [hunk 4]
```diff
@@ -72,7 +72,29 @@ See [Communicate when Slack is unavailable](/manual/slack-unavailable.html#2nd-l
...
+```
+@here this week we're going to be drilling deploying from AWS CodeCommit without using GitHub on 2ndline.
+Part of this drill involves disabling CD for all applications in all environments for ~20 minutes.
+We're planning on actioning the step of the documentation that disables auto-sync on Argo on <Wednesday at 2.30pm>.
+
+Please let us know if this timing will cause you any issues and we can reschedule.
+If there's no objections to the time i'll update the channel shortly before disabling and re-enabling CD.
+```
+
+2. Half an hour before scheduled time, follow the [Deploy when GitHub is unavailable](/manual/github-unavailable.html#create-and-deploy-an-image-to-ecr) instructions. Stopping at Deploy image to ArgoCD step.
```
Comment (ChrisBAshton) [inline source/manual/2nd-line-drills.html.md:96]:
```suggestion
2. Half an hour before scheduled time, follow the [Deploy when GitHub is unavailable](/manual/github-unavailable.html#create-and-deploy-an-image-to-ecr) instructions, stopping at the "Deploy image to ArgoCD step".
```

File: source/manual/2nd-line-drills.html.md [hunk 5]
```diff
@@ -72,7 +72,29 @@ See [Communicate when Slack is unavailable](/manual/slack-unavailable.html#2nd-l
 
 ### Deploy from AWS CodeCommit when Github is unavailable
 
-Follow the [Deploy when GitHub is unavailable](/manual/github-unavailable.html#create-and-deploy-an-image-to-ecr) instructions.
+#### Warnings
+
+- Please run the following drill in `integration`. Some steps in the guide below refer to production poweruser, be mindful and choose the correct environment for the drill.
+- Please choose a low impact deployable application that has a Dockerfile. The drill does not cater for repos that are dependencies such as gems and npm packages.
+- This drill requires a pause of Continuous Deployment for __all applications__ deployed through ArgoCD for about 20 minutes.
```
Comment (ChrisBAshton) [inline source/manual/2nd-line-drills.html.md:79]: I'd suggest allowing longer (1 hr?), to take the pressure off.

**Review by ChrisBAshton — COMMENTED** [review]
LGTM - some comments below.

File: source/manual/github-unavailable.html.md (hunk 1 above)
Comment (minhngocd) [reply source/manual/github-unavailable.html.md:21]: I've added a comment about this within the Clone from CodeCommit guide.

File: source/manual/2nd-line-drills.html.md (hunk 3 above)
Comment (minhngocd) [reply source/manual/2nd-line-drills.html.md:78]: Just wanted to pop this on as an explicit callout to **not** choose gems/npm cause we went for govuk-admin-template, thinking it's the lowest impact 🙈

**Pull request merged**